extensions/wikia/ArticleVideo/ArticleVideoContext.class.php

```diff
@@ -146,4 +146,24 @@ public static function isRecommendedVideoAvailable( int $pageId ): bool {
 			!WikiaPageType::isActionPage() &&
 			empty( self::isFeaturedVideoEmbedded( $pageId ) );
 	}
+
+	public static function getRelatedMediaIdForRecommendedVideo(): string {
+		$wg = F::app()->wg;
+
+		$relatedMediaId = [
```

Comment:
If we hardcode it, we will not be able to add new communities to the test in WikiFactory (I mean it will work, but newly added communities will always get the default mediaId). Maybe we could move it to WF?